install/ocp-fractions-scc.yaml

```diff
@@ -50,3 +50,14 @@ runAsUser:
   type: RunAsAny
 seLinuxContext:
   type: RunAsAny
+---
+apiVersion: v1
```

Comment:
These files contain only `k8s` and `ocp` stuff and we send these files for review for permissions - i think it's better to add this configmap on the installkit, and set some value on the `runaiconfig` so it will apply